src/behaviors/scrollable.js

```diff
@@ -13,11 +13,15 @@ export default function scrollable(WrappedComponent) {
     }
 
     componentDidMount() {
-      this.node.addEventListener('scroll', this.updateScrollState);
+      if (this.node) {
```

Comment:
You know, I don't run into a problem running the code. But once I added "scrollable" to Menu, I get an error with the Menu test: 
```
● Menu component › renders menu with list

  TypeError: Cannot read property 'addEventListener' of null

    at Scrollable.componentDidMount (src\behaviors\scrollable.js:33:16)
```

If this is not really needed, I can take it out. But I wasn't familiar enough with what it is doing to make that call.